Source/Libraries/GSF.Web/Model/ModelController.cs

```diff
@@ -517,37 +507,23 @@ public virtual IHttpActionResult GetPagedList([FromBody] PostData postData, int
 
         #region [Helper Methods]
 
-        protected string BuildWhereClause(IEnumerable<Search> searches)
+        protected string BuildWhereClause(IEnumerable<SQLSearchFilter> searches, List<object> parameters)
         {
+            List<string> clauses = [];
 
-            string whereClause = string.Join(" AND ", searches.Select(search => {
-                bool isQuery = false;
-                string searchText = search.SearchText;
-                if (searchText == string.Empty) searchText = "%";
-                else searchText = searchText.Replace("*", "%");
+            foreach (SQLSearchFilter search in searches)
+            {
+                if (search.SearchText == string.Empty)
+                    search.SearchText = "%";
```

Comment:
Perhaps this only makes sense when the operator is LIKE/NOT LIKE?

Also, the original logic supported asterisks. Are we doing away with that?

Reply:
Good Catch it's missing `search.SearchText= search.SearchText.Replace("*", "%");

Reply:
Then what do you think of this? or do we need to keep the old logic and ignore the operator?

```suggestion
                if (search.Operator == "LIKE" || search.Operator == "NOT LIKE")
                {
                    if (search.SearchText == string.Empty)
                        search.SearchText = "%";
                    else
                        search.SearchText = search.SearchText.Replace("*", "%");
                }
```